actors/runtime/src/main/java/cloud/orbit/actors/runtime/Hosting.java

```diff
@@ -88,13 +92,15 @@
 
     private CompletableFuture<Void> hostingActive = new Task<>();
 
-    private int maxLocalAddressCacheCount = 10_000;
-
     private NodeSelectorExtension nodeSelector;
 
-    public Hosting()
+    public Hosting(final int maxLocalAddressCacheCount, final long defaultActorTTL)
```

Comment:
To align with the actual parameter being used in the builder I would say the `maxLocalAddressCacheCount` is renamed to `localAddressCacheMaximumSize`

The parameter shouldn't be labelled as the `defaultActorTTL` in the context of this class. It should be something closer to `localAddressCacheTTL`.